src/homekit.cs

```diff
@@ -598,6 +602,10 @@ partial interface HMHome {
 		[Export ("addAndSetupAccessoriesWithCompletionHandler:")]
 		void AddAndSetupAccessories (Action<NSError> completion);
 
+		[NoWatch, NoTV, iOS (11,3)]
+		[Export ("addAndSetupAccessoriesWithPayload:completionHandler:")]
+		void AddAndSetupAccessories (HMAccessorySetupPayload payload, Action<NSArray, NSError> completion);
```

Comment:
`Action<NSArray, NSError> completion` => NSArray? We need a better type here :)

Reply:
The information is available `NSArray<HMAccessory *>`
so `HMAccessory[]` should be used